whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -74,6 +76,9 @@
 
     private static final List<String> RDAP_CONFORMANCE_LEVEL = Lists.newArrayList("rdap_level_0");
 
+    public static final String NOICE_MULTIPLE_VALUE_DESC = "There are multiple %s %s in %s, but only the first %s %s was returned.";
+    public static final String NOTICE_MULTIPLE_VALUE_TITLE = "Multiple %s found";
```

Comment:
No need for a static field if the value is only used once? It's more readable if the text is where it's used